lib/kontena/machine/digital_ocean/cloudinit.yml

```diff
@@ -48,6 +52,12 @@ coreos:
         Name=weave*
```

Comment:
Makes sense, I'll try without the `10-weave.network`. With the `10-weave.network` `networkctl` shows that `weave` is managed & routable.. we probably want it to be unmanaged.